File: Conditionals/Time_of_Day.c
// Sasha Rinitra, Template
#include <stdio.h>
#include <time.h>

int main(void) {

    time_t now = time(NULL);
    struct tm *tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;
    if (hour >= 12 && hour <= 16){
        printf("Good Afternoon!");
    }else if(hour > 16 && hour <= 19){
        printf("Good Evening!!");
    }else if(hour > 19){
        printf("Good night!!");
    }else{
        printf("Good morning!!");
    }
  
  return 0;
}

File: Conditionals/Time of Day.py
#Sasha Rinitra, Time of day Python

import time

current = time.time()
now = time.ctime(current)
local_time = time.localtime(current)
hour = local_time.tm_hour

if hour >= 12 and hour <= 16:
    print("Good afternoon!!")
elif hour > 16 and hour <= 19:
    print("Good Evening!!")
elif hour > 19:
    print("Good night!!")
else:
    print("Good morning!!")